src/Lib/Dispatcher.php

```diff
@@ -138,7 +138,7 @@ public static function dispatch($requestUri, $isCli = false)
         }
 
         $result = null;
-        if (method_exists($ctrl, $action)) {
+        if (method_exists($ctrl, $action) || is_callable([$ctrl, $action])) {
```

Comment:
Does it makes sense to just use 'is_callable'? Seems like that covers method_exists.